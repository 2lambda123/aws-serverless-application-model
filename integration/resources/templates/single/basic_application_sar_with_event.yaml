Resources:
  NestedApp:
    Type: AWS::Serverless::Application
    Properties:
      Location:
        ApplicationId: 'arn:aws:serverlessrepo:us-east-1:380013960170:applications/test-app-for-testing'
        SemanticVersion: 1.0.1
      Events:
        APILambda:
          Type: S3
          Properties:
            Bucket:
              Ref: Images
            Events: s3:ObjectCreated:*    

  Images:
    Type: AWS::S3::Bucket
