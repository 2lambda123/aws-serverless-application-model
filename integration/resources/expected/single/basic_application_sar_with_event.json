[
    { "LogicalResourceId":"Images", "ResourceType":"AWS::S3::Bucket" },
    { "LogicalResourceId":"NestedApp", "ResourceType":"AWS::CloudFormation::Stack" },
    { "LogicalResourceId":"NestedAppAPILambdaPermission", "ResourceType":"AWS::Lambda::Permission" }
]
